webapplication/user/forms.py

```diff
@@ -0,0 +1,57 @@
+from dj_rest_auth.forms import AllAuthPasswordResetForm
+from dj_rest_auth.app_settings import api_settings
+from django.conf import settings
+from django.contrib.sites.shortcuts import get_current_site
+from django.urls import reverse
+
+if 'allauth' in settings.INSTALLED_APPS:
+    from allauth.account import app_settings as allauth_account_settings
+    from allauth.account.adapter import get_adapter
+    from allauth.account.forms import default_token_generator
+    from allauth.account.utils import (
+        user_pk_to_url_str,
+        user_username,
+    )
+    from allauth.utils import build_absolute_uri
```

Comment:
Use try...except. In except clause define PasswordResetForm to be DefaultPasswordResetForm. Thus the rest of the code doesn't contain logic like `if 'allauth' in settings.INSTALLED_APPS`